moveit_core/robot_trajectory/include/moveit/robot_trajectory/robot_trajectory.h

```diff
@@ -204,7 +212,7 @@ class RobotTrajectory
 
   double getAverageSegmentDuration() const;
 
-  void getRobotTrajectoryMsg(moveit_msgs::msg::RobotTrajectory& trajectory) const;
+  void getRobotTrajectoryMsg(moveit_msgs::msg::RobotTrajectory& trajectory);
```

Comment:
Why is this no longer const?

Reply:
@mlautman causes issues with clang in OS X. I had to track this to one of our previous discussions, see https://github.com/AcutronicRobotics/moveit2/pull/8 or directly https://www.boost.org/doc/libs/1_64_0/libs/bind/doc/html/bind.html#bind.troubleshooting.const_in_signatures for a recommendation.